addons/project/models/project.py

```diff
@@ -507,6 +544,16 @@ def action_view_tasks_analysis(self):
         action_context['search_default_project_id'] = self.id
         return dict(action, context=action_context)
 
+    @api.model
+    def _action_open_all_projects(self):
+        action = self.env['ir.actions.act_window']._for_xml_id('project.open_view_project_all')
+        if self.user_has_groups('project.group_project_stages'):
+            action_context = ast.literal_eval(action['context']) if action['context'] else {}
+            action_context.update(self._context)
+            action_context['search_default_groupby_stage'] = True
```

Comment:
I don't think we _need_ it but since it is an action I thought it would be good to keep the context in case some defaults were set or something like that

Reply:
Ok, thanks 🙂

Reply:
I did apply your nice little recommendation though 😄